tests/process.rs

```diff
@@ -416,55 +416,115 @@ fn test_refresh_process_doesnt_remove() {
     not(feature = "unknown-ci")
 ))]
 fn test_refresh_tasks() {
+    // Skip if unsupported.
```

Comment:
Pretty nice improvement of the test. :+1:

Reply:
Thanks. I try to avoid sleeps in test as much as possible.